.example.env

```diff
@@ -11,6 +11,8 @@ AWS_REGION=us-east-1
 AWS_ENDPOINT=http://localhost:8000
 AWS_DYNAMO_LOCAL_PORT=8000
 AWS_DYNAMO_ADMIN_PORT=8001
+AWS_LOCAL=false
```

Comment:
within .env.dev with this value, and for prod with true ? correct ?

Reply:
you are right